doc/man/man3/crypto_kex_xk1_init_client.3monocypher

```diff
@@ -84,44 +84,63 @@ These functions differ from the
 functions in that the functions described in this page provide stronger
 security guarantees, but also require the server to be able to respond
 to a connection from a client immediately.
-.Ss Key exchange as the client
-When starting a new connection to the server, the client first calls
-.Fn crypto_client_xk1_init_client
-to initialise a
-.Vt crypto_kex_ctx .
-The arguments to
-.Fn crypto_client_xk1_init_client
-are:
+.Pp
+The arguments are:
 .Bl -tag -width Ds
-.It Fa ctx
+.It Fa client_ctx
+A pointer to a
+.Vt crypto_kex_client_ctx ,
+i.e. the context for the client.
+.It Fa server_ctx
 A pointer to a
-.Vt crypto_kex_ctx .
+.Vt crypto_kex_server_ctx ,
+i.e. the context for the server.
 .It Fa random_seed
 32 randomly generated bytes.
+The server and the client must
+.Em each
+generate a
+.Fa random_seed .
 See
 .Xr intro 3monocypher
 for advice about generating random bytes (use the operating system's
 random number generator).
 This argument is not constant because it is wiped after use.
-.It Fa local_sk
+.It Fa client_sk
 A 32-byte secret random number.
-This is a long-term secret used to identify the respective party.
-.It Fa local_pk
+This is a long-term secret used to identify the client.
+.It Fa client_pk
 The public key corresponding to the
-.Fa local_sk ,
+.Fa client_sk ,
 generated using
 .Xr crypto_key_exchange_public_key 3monocypher .
 If
 .Dv NULL ,
 the public key will be generated for you.
-.It Fa remote_pk
+.It Fa server_sk
+A 32-byte secret random number.
+This is a long-term secret used to identify the server.
+.It Fa server_pk
 The public key of the server, which must be known to the client before
 making a connection.
 This is used to authenticate the server's identity.
+It must thus be known to the client before making a connection.
+.It Fa msg1
+A 32-byte message, which is generated and sent by the client.
```

Comment:
I'll correct the network amplification problem ASAP. Monokex is undergoing a [significant change](http://loup-vaillant.fr/projects/xckdf) right now, so it will take a little while.